draft-ietf-httpbis-digest-headers.md

```diff
@@ -335,49 +335,49 @@ some algorithms, one or more parameters can be supplied.
 ~~~
 
 The BNF for "parameter" is defined in Section 5.4.1.4 of
-{{SEMANTICS}}. All digest-algorithm values are case-insensitive.
+{{SEMANTICS}}. All digest-algorithm values are case-insensitive but SHOULD be used in lowercase.
 
```

Comment:
TBH, I'm not sure how a SHOULD is justified here. This is not using SF syntax, so why does it matter?

Reply:
Which is a sensible way to signal we want to transition to lowercase and in future releases of Digest we may want to switch to SF ?

Reply:
@reschke I used `preferred`. WDYT?